.github/hub/push_evaluations_to_hub.py

```diff
@@ -13,6 +13,23 @@
 GIT_UP_TO_DATE = "On branch main\nYour branch is up to date with 'origin/main'.\
 \n\nnothing to commit, working tree clean\n"
 
+
+def get_git_tag(lib_path, commit_hash):
+    command = f"git describe --exact-match {commit_hash}"
```

Comment:
Maybe explain why this works or add link to StackOverflow :smile:

Reply:
done :)